backend/src/routes/payments.js

```diff
@@ -42,22 +69,43 @@ const paymentReturn = async (req, res) => {
         const ordernumber = req.query.ORDER_NUMBER;
         const payment = await db.payments.getPaymentByOrderNumber(ordernumber);
         if (!payment) {
-            return res
-                .status(422)
-                .json('Payment details not available in the system.');
+            console.log('Payment details not available in the system.');
+            return res.redirect(`/payment-complete?status=1`);
+        }
+
+        if (payment.payment_status) {
+            console.log('This payment was already processed!.');
```

Comment:
It would be good that the log messages would include some identifier for which payment this has happened like some kind of payment id. Also I think that this is an error and should be logged with `console.error`.

Reply:
corrected